css/webstyle.css

```diff
@@ -0,0 +1,346 @@
+  line-height: 24px;
+  font-weight: 600;
+  padding-right: 7%;
+}
+
+.work :nth-last-child(-n+2) {
+  color: #7a869a;
+  list-style: disc;
+  padding-left: 0;
+}
+
+ul .languages {
+  color: #6070ff;
+  background-color: #ebebff;
+  border-radius: 8px;
+  text-align: center;
+  padding: 4px 12px;
+  margin-bottom: 0;
+  display: block;
+}
+
+form {
+  align-self: stretch;
+  margin-bottom: 5%;
+  height: 100%;
+}
+
+#hand {
+  background-image: url(../images/hand-cur.png);
+  background-repeat: no-repeat;
+  background-position: 90% 20%;
+}
+
+button {
+  color: #396df2;
+  margin-top: 12px;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  text-align: center;
+  padding: 12px;
+  font-weight: 500;
+  width: 124px;
+  line-height: 24px;
+  font-size: 17px;
+  letter-spacing: 0.3px;
+  background-color: #fff;
+}
+
+.aboutme {
+  display: grid;
+  grid-template-rows: repeat(3, auto);
+  grid-template-columns: 42% 50%;
+  background-color: #fff;
+  border-top-right-radius: 8.5% 8.5%;
+  margin-top: 9.97%;
+  padding-top: 9.97%;
+}
+
+.abouttxt {
+  grid-row: 1;
+  grid-column: 1;
+  padding-left: 0;
+  margin-left: 24%;
+}
+
+#getbutton {
+  width: 162px;
+}
+
+#aboutlang {
+  grid-row: 1;
+  grid-column: 2;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+}
+
+h6 {
+  width: 100%;
+  font-size: 20px;
+  font-weight: 500;
+  line-height: 24px;
+  display: flex;
+  justify-content: space-between;
+  padding-right: 15px;
+  margin-block-end: 0;
+}
+
+#vertical {
+  writing-mode: vertical-rl;
+  text-orientation: mixed;
+  align-self: baseline;
+}
+
+span {
+  padding-right: 2%;
+}
+
+.langicons {
+  padding: 76px 12px 20px 4%;
+  margin-top: 8%;
+  width: 122px;
+  background-repeat: no-repeat;
+  background-color: #f7f7f9;
+  background-position: 12% 12%;
+  border-radius: 8px;
+}
+
+#js {
+  background-image: url(../images/JS.png);
+}
+
+#hm {
+  background-image: url(../images/HTML.png);
+}
+
+#cs {
+  background-image: url(../images/CSS.png);
+}
+
+.underline {
+  border-bottom: 1px solid #dfe1e6;
+}
+
+footer {
+  grid-row: 3;
+  grid-column: 1/span3;
+  display: flex;
+  flex-direction: column;
+  background-color: #6070ff;
+  background-image: url(../images/contact_background_shapes.png);
+  background-size: 90%;
+  background-repeat: no-repeat;
+  background-position: 26%;
+  border-top-right-radius: 8.5%;
+}
+
+.contact {
+  align-self: center;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+h5 {
+  color: #fff;
+  text-align: center;
+  font-size: 40px;
+  line-height: 52px;
+  font-weight: 700;
+  margin-top: 8.9%;
+  margin-block-end: 0;
+}
+
+#conttext {
+  color: #ebebff;
+  width: 40%;
+  text-align: center;
+}
+
+.input {
+  width: 120%;
+  margin: 3.5% 0;
+  padding: 15px 38px 15px 16px;
+  line-height: 20px;
+}
+
+textarea {
+  resize: none;
+}
+
+input::placeholder,
+textarea::placeholder {
+  font-family: "Poppins", sans-serif;
+  font-size: 20px;
+  color: #172b4d;
+}
```

Comment:
- [ ] The form is not correctly set, and I don't know why you have 3 CSS files.

- **Figma template**

![form](https://user-images.githubusercontent.com/61562626/134772236-c7ae6718-6b89-4756-a480-a932dc95a8a5.jpeg)


- **Your project**

![your_form](https://user-images.githubusercontent.com/61562626/134772244-765e2c00-7bca-44e9-9f43-878f0b672650.jpeg)

Reply:
screen shot from index.html